monitoring/uptime/uptime.go

```diff
@@ -62,9 +63,49 @@ func create(w io.Writer, projectID string) (*monitoringpb.UptimeCheckConfig, err
 	}
 	config, err := client.CreateUptimeCheckConfig(ctx, req)
 	if err != nil {
-		return nil, fmt.Errorf("CreateUptimeCheckConfig: %v", err)
+		return nil, fmt.Errorf("CreateUptimeCheckConfig GET: %v", err)
 	}
-	fmt.Fprintf(w, "Successfully created uptime check %q\n", config.GetDisplayName())
+	fmt.Fprintf(w, "Successfully created GET uptime check %q\n", config.GetDisplayName())
+	return config, nil
+}
+
+// createPost creates an example uptime check on a POST request.
+func createPost(w io.Writer, projectID string) (*monitoringpb.UptimeCheckConfig, error) {
+	ctx := context.Background()
+	client, err := monitoring.NewUptimeCheckClient(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("NewUptimeCheckClient: %v", err)
+	}
+	defer client.Close()
+	req := &monitoringpb.CreateUptimeCheckConfigRequest{
+		Parent: "projects/" + projectID,
+		UptimeCheckConfig: &monitoringpb.UptimeCheckConfig{
+			DisplayName: "new POST uptime check",
+			Resource: &monitoringpb.UptimeCheckConfig_MonitoredResource{
+				MonitoredResource: &monitoredres.MonitoredResource{
+					Type: "uptime_url",
+					Labels: map[string]string{
+						"host": "example.com",
+					},
+				},
+			},
+			CheckRequestType: &monitoringpb.UptimeCheckConfig_HttpCheck_{
+				HttpCheck: &monitoringpb.UptimeCheckConfig_HttpCheck{
+					RequestMethod: monitoringpb.UptimeCheckConfig_HttpCheck_POST,
+					ContentType:   monitoringpb.UptimeCheckConfig_HttpCheck_URL_ENCODED,
```

Comment:
How about adding a body like `foo=bar`? The field is type bytes so it might be worth doing whatever conversion the language requires.

Reply:
Agreed, considering it was non-trivial for me to figure out how to do it, especially for the URL-encoded content type.